tests/unit/src/test/scala/tests/DocumentSymbolSlowSuite.scala

```diff
@@ -0,0 +1,92 @@
+package tests
+
+object DocumentSymbolSlowSuite extends BaseSlowSuite("documentSymbol") {
+
+  testAsync("documentSymbol") {
```

Comment:
nit, I usually begin with a `"basic"` test suite when I have no idea for the name

Reply:
this test case stresses error handling however, so I would name it `"error"` instead